node/stream.sh

```
@@ -27,26 +27,42 @@ mkdir -p /tmp/$NODE_NAME/hls/$timestamp
+    NICE_HIGH="nice -n -10"
+    NICE_MED="nice -n -7"
+    JACKD_RT="-P 75"
+else
+    echo "Note: cannot set elevated process priority (CAP_SYS_NICE not available); running at default priority"
+    NICE_HIGH=""
+    NICE_MED=""
+    JACKD_RT=""
+fi
+
+#  Setup jack
+JACK_NO_AUDIO_RESERVATION=1 jackd -t 2000 $JACKD_RT -d alsa -d hw:$AUDIO_HW_ID -r $SAMPLE_RATE -p 1024 -n 10 -s &
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)